lca-cli/installation_configuration_files/services/installation-configuration.service

```diff
@@ -6,6 +6,8 @@ After=prepare-installation-configuration.service NetworkManager-wait-online.serv
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=/usr/local/bin/lca-cli post-pivot
+ExecStopPost=-/usr/local/bin/lca-cli init-monitor --exec-stop-post postpivot
```

Comment:
Currently this service is built in a way that it should restart on failure and failure is possible. We can't add this line till post-pivot logic is not changed in case you want to handle roll-back on service failure.
From my personal opinion this logic should be added to post pivot itself. In case there is an error we can verify how much time we are running and run a rollback. Currently this change will break pretty many installations as for example "applying manifest can fail and it is ok" or "pulling image can fail and it is ok" .

Reply:
Talked about it, we can resolve the comment